packages/excalidraw/components/App.tsx

```diff
@@ -1529,7 +1529,11 @@ class App extends React.Component<AppProps, AppState> {
               : FRAME_STYLE.nameColorLightTheme,
             lineHeight: FRAME_STYLE.nameLineHeight,
             width: "max-content",
-            maxWidth: `${f.width}px`,
+            maxWidth: this.state.searchMatches.find(
+              (sm) => sm.id === f.id && sm.focus,
+            )
+              ? "none"
+              : `${f.width * this.state.zoom.value}px`,
```

Comment:
Should we prefilter frame matches into a Map so we don't have to do lookups for each visible frame like this. Though downside is we'd be prefiltering into a Map every time even if no frames were currently visible in the viewport.

Reply:
I've assigned `const focusedSearchMatch = this.state.searchMatches.find((sm) => sm.focus);` outside the frames loop. For each frame in the frame, it's then `focusedSearchMatch?.id === f.id && focusedSearchMatch?.focus`.